pkg/pgmodel/end_to_end_tests/promql_endpoint_integration_test.go

```diff
@@ -358,24 +510,9 @@ func testRequest(t *testing.T, req *http.Request, handler http.Handler, client *
 		}
 		defer tsResp.Body.Close()
 
-		var got, wanted response
-
-		err = json.Unmarshal(tsContent, &got)
+		err = comparator(promContent, tsContent)
```

Comment:
should we also be comparing headers in this function? Not just the body? (ok to leave to another PR but let's file an issue if we do)

Reply:
added to project board